src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -0,0 +1,22 @@
+package org.zalando.nakadi.service.timeline;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.zalando.nakadi.domain.Timeline;
+import org.zalando.nakadi.repository.db.TimelineDbRepository;
+
+import java.util.List;
+
+@Service
+public class TimelineService {
```

Comment:
That is true, but there is a ticket called "CRUD for timelines", so I think this service will be needed anyway.